lib/Controller/PageController.php

```diff
@@ -69,13 +71,19 @@ public function __construct(string $appName,
 	 * Default routing
 	 */
 	public function index(): TemplateResponse {
+		$userId = \OC_User::getUser();
```

Comment:
Please inject it, do not use this :)

https://github.com/nextcloud/server/blob/5bf3d1bb384da56adbf205752be8f840aac3b0c5/lib/public/IUserSession.php#L79